### PR TITLE
fix(curriculum): add result of comparison expression evaluation to the inline comment

### LINKED ISSUE #45183: light visual/order improvement to the explanation of content
## Affected page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator

## Your code

```

no code


```

## Expected behavior

The student has to go all the way down up to the explanation "In order, these expressions would evaluate to true, false, true, and true." to realize what the result of the operations is.
My suggestion is that the second section of the code  should look like this:

1   ==  1    // true
1   ==  2    // false
1   == '1'   // true
"3" ==  3   // true

instead of the screenshot attached.

## Screenshots

image

## System

- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

### PR DESCRIPTION
**Pull request by HenMoshe**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #45183

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md [hunk 1]
```diff
@@ -25,14 +25,12 @@ function equalityTest(myVal) {
...
 
 ```js
-1   ==  1
-1   ==  2
-1   == '1'
-"3" ==  3
+1   ==  1 // true
+1   ==  2 // false
+1   == '1' // true
+"3" ==  3 // true
```
Comment (nhcarrigan) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md:31]:
```suggestion
1   ==  1  // true
1   ==  2  // false
1   == '1' // true
"3" ==  3  // true
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-or-equal-to-operator.md [hunk 2]
```diff
@@ -16,14 +16,12 @@ Like the equality operator, the greater than or equal to operator will convert d
...
 
 ```js
-6   >=  6
-7   >= '3'
-2   >=  3
-'7' >=  9
+6   >=  6 // true
+7   >= '3' // true
+2   >=  3 // false
+'7' >=  9 // false
```
Comment (nhcarrigan) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-or-equal-to-operator.md:22]:
```suggestion
6   >=  6  // true
7   >= '3' // true
2   >=  3  // false
'7' >=  9  // false
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Hey there,

It's a bit nit-picky, but can we go ahead and line the comments up vertically? Looks a bit cleaner that way.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md (hunk 1 above)
Comment (HenMoshe) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md:31]: Yea for sure! should i just commit suggestion from here? sorry never used this feature.

Comment (nhcarrigan) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md:31]: If you switch to the `files` tab, you can go through and hit "Add suggestion to batch" on each of them - that'll let you commit them all at once.

**Review by naomi-lgbt — APPROVED** [review]
This looks good to me!

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md (hunk 1 above)
Comment (HenMoshe) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md]: thanks! have a good day

**Review by gikf — APPROVED** [review]
Looks good! :+1:

**Comment by gikf** [comment]
Hi @HenMoshe. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**